.github/workflows/release.yml

```diff
@@ -673,11 +674,8 @@ jobs:
         uses: actions/download-artifact@v1
         with:
           name: oso_js_node
-      - name: Run tests
-        # Timeout fix: https://stackoverflow.com/a/51508426
-        run: |
-          yarn install --prod  --network-timeout 100000
```

Comment:
I dont understand why we don't need to install dependencies here?

Reply:
There are no dependencies to install — Oso has zero runtime JS dependencies, and the built parity test in `oso_js_node` imports `Oso` & `Variable` from the built Oso in `oso_js_node` and `join` from Node's `path` module and then loads the `test.polar` file via a relative path.

Reply:
Ahhh, got it. Okay